lib/gscraper_web/templates/registration/new.html.eex

```diff
@@ -1,27 +1,30 @@
 <div class="card card-auth">
   <div class="card-auth__title">
-    <h4><%= dgettext "auth", "Sign up" %></h4>
+    <h4><%= dgettext("auth", "Sign up") %></h4>
   </div>
-  <form class="form form-auth">
+
+  <%= form_for @changeset, Routes.registration_path(@conn, :create), [class: "form form-auth"], fn f -> %>
     <div class="form__group required">
-      <label for="username" class="required">Username</label>
-      <input class="form__control" required="required" autofocus="autofocus" type="text" name="email">
-      <div class="form__feedback"></div>
+      <%= label f, :username %>
```

Comment:
Just wondering if these helpers play well with gettext and which key it fetches the text from. I haven't used it much. 😆 💭

Reply:
according to the [Form documentation](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Form.html#label/2), we can specify the text for the label, thus using `gettext` like this:
```elixir
<%= label f, :username, gettext("Username") %>
```
I added it in 0b23901